src/overlays/actors/ovl_En_Gm/z_en_gm.c

```diff
@@ -1486,21 +1567,20 @@ void func_80950C24(EnGm* this, GlobalContext* globalCtx) {
 }
 
 void func_80950CDC(EnGm* this, GlobalContext* globalCtx) {
-    u32* unk_14 = &gSaveContext.unk_14;
-    struct_80133038_arg2 sp20;
+    ScheduleResult sp20;
 
-    this->unk_3C4 = REG(15) + *unk_14;
```

Comment:
There are several instances of this pattern (it had to be changed when the size of `ScheduleResult` increased). Should this be a macro?

Reply:
What's the pattern exactly?
`REG(15) + ((void)0, gSaveContext.unk_14)`?

Reply:
I recall being told that some accesses to fields in the save struct were done like this. I believe the thought was that this was a guard to protect against accidentally writing to it (as `((void)0, gSaveContext.unk_14)` results in an r-value). An example of it in current code is the [`CURRENT_DAY` macro](https://github.com/zeldaret/mm/blob/850823c2ee89aad0f0e53aa70777af11af6d5433/include/macros.h#L51).

Reply:
Hmmm, yeah we should probably make macros for this kind of stuff, but a change like that should probably be a PR on its own and may require some discussion.